spec/fortnox/api/validators/context.rb

```diff
@@ -13,7 +13,7 @@
   end
   shared_examples_for 'invalid' do |attribute, values, error_type|
     values.each do |value|
-      context "when #{attribute} set to #{value}" do
+      context "when :#{attribute} set to #{value}" do
```

Comment:
Yes, I know. You are printing it as: `"This is an :attribute"`, which is fine as long as we know we will always get a symbol. If you do `"This is an #{ attribute.inspect }"` it would still be `"This is an :attribute"` if the `attribute` variable contained a symbol but would be `"This is an \"attribute\""` if the attribute var was a string instead etc ...